eve-api-data/src/main/java/com/eveonline/api/config/RepositoryConfiguration.java

```diff
@@ -1,15 +1,15 @@
 package com.eveonline.api.config;
 
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.orm.jpa.EntityScan;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
+import com.eveonline.api.repo.ApiDataRepositoryPackage;
+
 @Configuration
```

Comment:
I love to hate that you added an @Configuration annotation here.  I love it because it's a good hidden way of getting things involved in the autoconfiguration that is the heart of spring-boot but I hate it at the same time because of that.  Due to how spring-boot autoscans things this might not be picked up and used as a configuration class.  The majority of this application will be in the com.aba or com.lucrh package with a package scan configuration the same and therefore this will never be picked up unless explicitly referenced.